src/app/game-state/activity.service.ts

```diff
@@ -689,27 +739,31 @@ export class ActivityService {
         if (numBuilders < 10){
           this.logService.addLogMessage("You fumble without the proper help and hurt yourself.","INJURY","EVENT");
           this.characterService.characterState.status.health.value -= this.characterService.characterState.status.health.max * 0.05;
+          this.mainLoopService.pause = true;
```

Comment:
Is this to help you debug? We for sure don't want to pause on every failure without some kind of checkbox to turn it off.

Reply:
This is a deadly fail though. 20 days goes by and suddenly you wonder what happened. Though I do agree power should be in the players' hands in this, I've already messed with quite a bit as you've said.